README.md

```diff
@@ -0,0 +1,31 @@
+# An Evolutionary Game Theoretic Model of Rhino Horn Devaluation
+
+**Authors**: @Nikoleta-v3 @drvinceknight @TamsinELee
+
+This repository for the source code for a paper accepted for publication in
```

Comment:
A repository or This repository contains?

Reply:
:+1: